frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-and-nodes-step.tsx

```diff
@@ -225,9 +190,9 @@ const SelectedCapacityAndNodes: React.FC<SelectedCapacityAndNodesProps> = ({
             }
           />
         </GridItem>
-        <GridItem span={10} className="odf-capacity-and-nodes__select-nodes">
+        <GridItem span={10}>
           <SelectedNodesTable data={nodes} />
-          {!!nodes.length && <NodesDetails nodes={nodes} />}
+          {!!nodes.length && <SelectNodesTableFooter nodes={nodes} />}
```

Comment:
you can move this inside <SelectedNodesTable> just like <SelectNodesTable>

Reply:
done